vendor.mod

```diff
@@ -6,6 +6,12 @@ module github.com/docker/cli
 
 go 1.23.0
 
+replace (
+	// FIXME(thaJeztah): temporarily need to pin on commits, otherwise go modules won't resolve until these are tagged.
+	github.com/moby/moby/api => github.com/moby/moby/api v0.0.0-20250721205005-81caabae43aa
```

Comment:
Don't want to just put these in the require for now rather than use replace?

Reply:
That didn't want to work, because api/ and client/ don't have existing tags, so go modules then generates an invalid pseudo version and barfs out

Reply:
I can give it another try to see what the exact error was; there's some more PRs merged in moby that I can bump as well; otherwise we'll just have to keep the replace for now